virttest/guest_agent.py

```diff
@@ -725,3 +725,35 @@ def fsthaw(self, check_status=True):
                     raise
             return ret
         return -1
+
+    def openfile(self, args):
+        """
+        args:specify file's path and mode option when open file
+        """
+        cmd = "guest-file-open"
+        if self.check_has_command(cmd):
+            return self.cmd(cmd, args, debug="Ture", success_resp="False")
```

Comment:
If success_resp="False", then it self.cmd always returns "" and returning it from def openfile() is not useful